src/Bicep.Cli/Program.cs

```diff
@@ -144,6 +150,32 @@ private void BuildManyFilesToStdOut(IDiagnosticLogger logger, string[] bicepPath
                 writer.WriteEndArray();
             }
         }
+
+        private static string ReadFile(string path)
+        {
+            try
+            {
+                return File.ReadAllText(path);
+            }
+            catch (Exception exception)
+            {
+                // I/O classes typically throw a large variety of exceptions
+                // instead of handling each one separately let's just trust the message we get
```

Comment:
In the case where multiple files are being built, will it be clear from the exception message which file raised the problem?

Reply:
File not found, directory not found, access denied and permission related ones do include the full path to the file. Not sure about others that can be thrown.